.github/dependabot.yml

```diff
@@ -5,7 +5,7 @@
 
 version: 2
 updates:
-  - package-ecosystem: "" # See documentation for possible values
+  - package-ecosystem: "npm" # See documentation for possible values
```

Comment:
yarn ?

Reply:
according to the documentation, yarn is marked as npm option